### PR TITLE
Integration with bitswap

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,107 @@
...
+  }
+
+  getBlock (key, extension, callback) {
+    if (this.isOnline()) {
+      if (typeof extension === 'function') {
+        callback = extension
+        extension = undefined
+      }
+
+      this._bitswap.getBlock(key, callback)
```
Comment (daviddias) [inline src/index.js:56]: So, bitswap will check the repo if it already has the block? (Thought that logic would be here, but having bitswap doing the job is also fine)

Comment (daviddias) [inline src/index.js:56]: Can docs be improved along the way too, please?

Comment (dignifiedquire) [inline src/index.js:56]: > So, bitswap will check the repo if it already has the block?

Yes, I need that logic in bitswap to be sure that no inefficiencies happen, so no need to replicate it here.

Comment (dignifiedquire) [inline src/index.js:56]: > Can docs be improved along the way too, please?

@diasdavid done

**Pull request merged**